modules/sdk-coin-tia/src/lib/keyPair.ts

```diff
@@ -0,0 +1,25 @@
+import { KeyPairOptions } from '@bitgo/sdk-core';
+import { pubkeyToAddress } from '@cosmjs/amino';
+
+import { CosmosKeyPair } from '@bitgo/abstract-cosmos';
+
+/**
+ * Celestia keys and address management.
+ */
+export class TiaKeyPair extends CosmosKeyPair {
+  constructor(source?: KeyPairOptions) {
+    super(source);
+  }
+
+  /** @inheritdoc */
+  getAddress(): string {
+    const base64String = Buffer.from(this.getKeys().pub.slice(0, 66), 'hex').toString('base64');
+    return pubkeyToAddress(
+      {
+        type: 'tendermint/PubKeySecp256k1',
+        value: base64String,
+      },
+      'celestia'
+    );
+  }
+}
```

Comment:
Will this method be similar for the other chains as well, apart from the prefix ? 

If yes, could we put this method in a common class and pass in the prefix instead ?

Reply:
getAddress function is defined in the parent class thats why we can't add a parameter to it

Reply:
Yes deepak, that's correct. That is why I had to move this method out of abstract cosmos sdk.